paypro-gateways-woocommerce/includes/paypro/wc/gateway/ideal.php

```diff
@@ -32,9 +26,9 @@ public function payment_fields()
         $selected_issuer = $this->getSelectedIssuer();
 
         $html = '<select name="' . PayPro_WC_Plugin::PLUGIN_ID . '_issuer_' . $this->id . '">';
-        $html .= '<option value=""></option>';
+        $html .= '<option value="" hidden></option>';
 
-        foreach($ideal_issuers['issuers'] as $issuer)
+        foreach($ideal_issuers['issuers'] ?? [] as $issuer)
```

Comment:
We currently still support PHP versions lower than 7.0 which means we cannot use `??` operator.

Reply:
will be updated